src/app/modules/submission/encrypt-submission/encrypt-submission.controller.ts

```diff
@@ -730,6 +768,15 @@ const _createSubmission = async ({
   }
 
   // Send success back to client
+  // clear cookies to log out user if isSingleSubmission form
```

Comment:
Hm. Would it make sense to clear on every case, not just when it is a `isSingleSubmission` form?

Reply:
I recall that we want to simplify the cases, and having a login session be ended on every form submission might be simplier.

Reply:
hi @KenLSM, i am not entirely sure if we want this behavior to force logout the user each time even for non `isSingleSubmission` forms. 

the rationale is for `MyInfo`, the user needs to approve permission each time to prefill form data, hence force clearing done previously makes sense. 

however, for other modes of Singpass, it might be useful for the user to stay logged in to submit multiple submissions if possible. 

The second alternative way to look at this is that the same user should never need to submit the same form more than once (not always true but not always false either) 

Since the discussion on which of the above 2 is ideal is out of scope of my current nric task, i did not change the existing clearing cookie behavior. 

but if we are sure we want option 2, then i suppose i can clear login cookies for all cases. 

please let me know what your thoughts on this are

Reply:
I think we've decided to go with option 2, which is to always clear to simplify submissions for now. Makes it easier to rationalise when explaining to admins / users.

There's a counter point that can be made as well for preventing logins, where the form is displayed on a public kiosk (i.e., ipad). For that case, we should always log the respondent out, rather than to keep them logged in. Both ways seemed have sufficient usecase, but for the sake of Product Operations, I think we'll lean on the simplicity of explanation.

We do have something in mind that the team might explore, which is the citizen portal where we can properly tackle problems like this more cleanly.

Reply:
cmiiw, also thinking that it would make your task easier if we include it in this scope, as it would mean that we get to delete code. Otherwise, agree to leave it out for a subsequent PR to sweep it out.

Reply:
deferring to future PR as discussed